include/tvm/runtime/packed_func.h

```diff
@@ -1377,7 +1449,7 @@ inline TObjectRef TVMPODValue_::AsObjectRef() const {
   using ContainerType = typename TObjectRef::ContainerType;
 
   if (type_code_ == kTVMNullptr) {
-    ICHECK(TObjectRef::_type_is_nullable)
+    CHECK(TObjectRef::_type_is_nullable)
```

Comment:
shouldn't this be static_assert?

Reply:
this should actually be a runtime check, note that constant folding will actually remove this completely

Reply:
I think this is out of scope for this PR.